WordPress/Classes/Services/BlogService.h

```diff
@@ -26,35 +26,6 @@ extern NSString *const WPBlogUpdatedNotification;
  */
 - (nullable Blog *)blogByHostname:(NSString *)hostname;
 
-/**
- Returns the blog currently flagged as the one last used, or the primary blog,
- or the first blog in an alphanumerically sorted list, whichever is found first.
- */
-- (nullable Blog *)lastUsedOrFirstBlog;
-
-/**
- Returns the blog currently flagged as the one last used, or the primary blog,
- or the first blog in an alphanumerically sorted list that supports the given
- feature, whichever is found first.
- */
-- (nullable Blog *)lastUsedOrFirstBlogThatSupports:(BlogFeature)feature;
```

Comment:
This method hasn't been converted. But the code builds without it, so it was clearly unused 😄 👍